CI/vagrant/start.sh

```diff
@@ -0,0 +1,7 @@
+#! /bin/bash
```

Comment:
Why do we need two "start.sh" files, one in root folder and the other in "iovnet" folder?